lib/Doctrine/DBAL/Connection.php

```diff
@@ -489,12 +489,14 @@ public function setAutoCommit($autoCommit)
      * Sets the fetch mode.
      *
      * @param integer $fetchMode
-     *
+     * @param mixed $arg2
```

Comment:
Also: not sure if I'd ever want these PDO horrors in here (it's something that we've deliberately left out also when building stuff like https://github.com/crate/crate-pdo, as this sort of API is kinda useless :-\ )